week3/server.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE `customer` (
+  `ID` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `f_name` varchar(35) NOT NULL,
+  `l_name` varchar(35) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone_number` varchar(45) NOT NULL,
+  `Address` varchar(255) NOT NULL
```

Comment:
Using all lowercase letters is better because some database management systems are case sensitive. If you ever need to move your database from MySQL to another database, this could introduce errors.